internal/provider/provider.go

```diff
@@ -69,6 +70,10 @@ func (p *ImageTestProvider) Metadata(ctx context.Context, req provider.MetadataR
 func (p *ImageTestProvider) Schema(ctx context.Context, req provider.SchemaRequest, resp *provider.SchemaResponse) {
 	resp.Schema = schema.Schema{
 		Attributes: map[string]schema.Attribute{
+			"host": schema.StringAttribute{
```

Comment:
is this needed? or is the inclusion of `client.FromEnv` enough for `DOCKER_HOST` to be parsed at runtime?

my assumption is we probably won't need to "hardcode" the host, and most of the use case is focused around setting it at runtime, but that may be wrong?

Reply:
It's not needed but I consider that a nice to have. If you set `DOCKER_HOST`, all Docker operations on that environment become bound to the host set on that environment variable, and I can see situations where when running locally you would like the provider to communicate with a specific host, whereas the CLI communicates with another one.

However I overlooked the bit where you're able to specify TLS credentials, so I'll need to update that part.